docs/network-flow-visibility.md

```diff
@@ -202,30 +203,69 @@ helm install theia build/charts/theia -n flow-visibility --create-namespace \
   --set=clickhouse.storageSize="2Gi"
 ```
 
-The ClickHouse TCP service is used by the Flow Aggregator. If you have changed the TCP port,
-please update the `databaseURL` following
-[Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1).
+The ClickHouse TCP Service is used by the Flow Aggregator. If you have changed
+the TCP port, please update the `clickHouse.databaseURL` in the Flow Aggregator
+Helm Chart values.
+
+The ClickHouse credentials are used in the Flow Aggregator. If you have changed
+the ClickHouse credentials, please update the `clickHouse.connectionSecret` in the
+Flow Aggregator Helm Chart values.
+
+Please refer to the Flow Aggregator Helm Chart
+[README](https://github.com/antrea-io/antrea/blob/main/build/charts/flow-aggregator/README.md)
+and [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
+to learn more about ClickHouse configuration options in Flow Aggregator.
+
+##### ClickHouse Cluster
+
+In v0.1, Theia deploys ClickHouse in a non-cluster mode, which cannot switch to
+ClickHouse cluster directly. You need to uninstall ClickHouse, delete related
+data if you use PersistenVolume and redeploy it when upgrading or downgrading
+between v0.1 and v0.2.
+
+Starting with v0.2, Theia supports ClickHouse database cluster with data replication,
+and deprecates the non-cluster mode. A ClickHouse cluster consists of one or more
```

Comment:
non-clustered mode is not deprecated, but removed.

Maybe we can simply state "Starting with v0.2, Theia supports ClickHouse clustering in lieu of the non-clustered mode supported in v0.1"

Reply:
Sounds good. Updated, thanks!